charts/keycloak/values.yaml

```
@@ -18,6 +18,22 @@ keycloak:
       standardFlowEnabled: true
       webOrigins:
       - +
+    - clientId: sigstore
```

```suggestion
    - clientId: trusted-artifact-signer
```

Recommended to align with official Red Hat product documentation

https://docs.redhat.com/en/documentation/red_hat_trusted_artifact_signer/1/html-single/deployment_guide/index